### ISSUE TITLE
grepl error when querying Russian wikipedia

### ISSUE THREAD

**Issue by erzk** [petermeissner/wikipediatrend#14]
I found another bug when querying Russian wikipedia.

I used the following code:
```
bkomoRU <- wp_trend(  page      = "Президент Польши", 
                      from      = "2015-01-01", 
                      lang      = "ru")
``` 
and I got the following error:
```
Error: grepl("\\w", page) is not TRUE
```

I'm not sure what's the reason for that.

**Comment by petermeissner** [comment]
Thanks for reporting!

I'll switch internals to ...
    grepl("\\w", "Президент Польши", useBytes = T)
    stringr::str_detect("Президент Польши", "\\w")

At the moment *wikipediatrend* is not well (i.e. not at all) equipped to handle special characters. But I work on it and it shall be solved in the next version patch.

**Comment by petermeissner** [comment]
solved since version 1.1.0
